pkg/controllers/multicluster/utils.go

```diff
@@ -134,36 +137,72 @@ func PortsEqualIgnoreOrder(a, b []*model.Port) (equal bool) {
 	return true
 }
 
+func IPsEqualIgnoreOrder(a, b []string) (equal bool) {
```

Comment:
Seems bit verbose. There is simpler solution mentioned here https://stackoverflow.com/a/67624073/791427

Reply:
Can use one of those options. Is it worth bringing in new imports?

Reply:
It's a standard google module, and quite small. I don't see concern. We possibly can use this for ports comparison too.